nokia/__init__.py

```diff
@@ -223,6 +248,9 @@ def is_measure(self):
 
     def is_target(self):
         return self.category == 2
+    
```

Comment:
Looks like a typo here. Also, this probably doesn't belong in this pull request

Reply:
Yes, you right, I was working with an older file.